installer/install.iss

```diff
@@ -2529,9 +2553,21 @@ begin
     DeleteMarkedEnvString('GIT_SSH');
     DeleteMarkedEnvString('SVN_SSH');
 
-    if (PuTTYPage<>NIL) and RdbSSH[GS_Plink].Checked then begin
-        SetAndMarkEnvString('GIT_SSH',EdtPlink.Text,True);
-        SetAndMarkEnvString('SVN_SSH',EdtPlink.Text,True);
+    if (PuTTYPage<>NIL) then begin
+        if RdbSSH[GS_OpenSSH].Checked then begin
```

Comment:
Shouldn't this be `if not RdbSSH[GS_Plink].Checked` instead? Just in case that future patches want to add a third option there?

Reply:
If another variant is added, isn't it likely to need to set ssh.variant (and env vars) to specific values?

How about:
    if (PuTTYPage<>NIL) then begin
        GitSystemConfigSet('ssh.variant',#0);
        if RdbSSH[GS_Plink].Checked then begin
If the putty page is present, clear and reset (if needed) the ssh.variant config value.  It may mean double calls to git config but ensures ssh.variant is cleared if there are any possible putty states and any future variants start with a clean slate.

Reply:
I like that idea!